package/gluon-hoodselector/luasrc/usr/sbin/hoodselector

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/lua
+
+local nixio = require('nixio')
```

Comment:
nixio was replaced by luaposix in gluon 2 weeks ago.
If a lib is used like that it should be a dependency in the makefile. In this case however I feel luaposix should be used.

Reply:
@2tata you removed the dependency, but didn't update this file here which still uses nixio

Reply:
Yes, I had remove it from hoodselector util, but need to think how to remove it from the hodselector it self.